lighthouse-cli/test/fixtures/dobetterweb/dbw_tester.html

```diff
@@ -182,6 +182,16 @@ <h2>Do better web tester page</h2>
 <!-- FAIL(efficient-animated-content): animated gif found -->
 <img src="lighthouse-rotating.gif" width="811" height="462">
 
+<!-- FAIL(tap-targts): buttons too close together -->
```

Comment:
this is a little weird because usually the PASS/FAIL markers are indicators for the specific smoke test that it's a pass/fail. `yarn smoke dbw` doesn't run `tap-targets`, so it wouldn't be an audit failure.

Buuuuuut, we've been using these test files in multiple tests for a while now (sometimes multiple smoke configs, and this one for `sample_v2.json`), so maybe it's fine you just have to cross reference the audit id with the config that's running the smoke test?

Reply:
Oh I see what you mean. Maybe the ultimate solution would be to separate the dbw smoke test from the sample page?